charts/hono/values.yaml

```diff
@@ -1708,7 +1711,8 @@ kafka:
       type: "pem"
       pemChainIncluded: true
       existingSecrets:
-      - "{{ .Release.Name }}-kafka-example-keys"
+      # Evaluated in kafka chart via tpl; kafka chart full name used here!
+      - "{{ include \"hono.fullname\" (dict \"dot\" .) }}-example-keys"
```

Comment:
if invoking the template works here, couldn't we also use something like this in order to prevent having to escape the quotes?
```yaml
- {{- printf "%s-example-keys" ( include "hono.fullname" (dict "dot" . ) ) | quote }}
```